src/test/java/ua/edu/internship/interview/service/business/SkillServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package ua.edu.internship.interview.service.business;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.service.dto.SkillTreeDto;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+
+import java.util.ArrayList;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.ArrayList'
<!-- __reviewdog__:ChBmMTc3OGE0OTFhZTRhNjg3EgpjaGVja3N0eWxl -->